src/libpmemset/pmemset.h

```diff
@@ -18,6 +20,14 @@ extern "C" {
 #define PMEMSET_LOG_LEVEL_VAR "PMEMSET_LOG_LEVEL"
 #define PMEMSET_LOG_FILE_VAR "PMEMSET_LOG_FILE"
 
+int pmemset_insert_part_map(struct pmemset *set, struct pmemset_part_map *map);
```

Comment:
These functions to me look like circular dependencies and probably shouldn't be needed.
`pmemset_source_destroy_pmem2_source` is not used anywhere.